_pages/en_US/troubleshooting.txt

```diff
@@ -155,3 +155,17 @@ Browser based exploits (such as browserhax or 2xrsa) are often unstable and cras
 1. Your device is bricked
 1. You will need to get an ntrboot-comptible flashcart (one of the ones on [this list](ntrboot) or a [hardmod](https://gbatemp.net/threads/414498/), or repair / replace your device
 1. For support (in English), join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp)
+
+## System update error after installing CFW
+
+ Occasionally, updates may fail to install after installing CFW. To fix this, reboot your 3DS after each step of this section, then try updating again.
+
+1. Set your DNS settings to "Auto"
+1. Move closer to your WiFi router
+1. Update from Safe Mode by turning off the console, holding (Left Shoulder) + (Right Shoulder) + (D-Pad Up) + (A) on boot, and following the on-screen prompts
+1. Delete your WiFi connection, then reconnect to your WiFi again
+1. Reboot your WiFi router
+1. Connect to a different WiFi connection, like a mobile hotspot
+1. Nintendo servers may be down. Try again later
```

Comment:
small nitpick but why do you say "[...] may be down. Try again later" then "[...], then try again" maybe using a comma on the former would be better

Reply:
It'd technically be grammatically incorrect because "try again later" is a full sentence. You can use a semicolon to separate them though.